packages/api/methods.test.ts

```diff
@@ -81,28 +81,22 @@ describe('API CRUD operations', () => {
     expect(groups).toEqual(
       expect.arrayContaining([
         expect.objectContaining({
-          hidden: 0,
```

Comment:
Updated since we're now (correctly) returning an `APICategoryGroupEntity` instead of a `CategoryGroupEntity`